examples/Reading in linearised models.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    mask[np.triu_indices_from(mask)] = True
</pre><p>Maybe use k=1 in triu_indices_from so that you get the main diagonal in the plots, as in <code>mask[np.triu_indices_from(mask, k=1)] = True </code>? The variance is as interesting as the covariances.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/gEconpy/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>